### DISCUSSION TITLE
upload_app_privacy_details_to_app_store with API KEY

### DISCUSSION THREAD

**Discussion by m-makhovyk** [fastlane/fastlane#18887] [Q&A] [answered]
So I decided to use App Store Connect Api Key and get rid of 2fa on our CI. But upload_app_privacy_details_to_app_store action requires username and I can't use api key with it. Isn't it supported yet? Or am I doing something wrong?

**Comment by crazymanish** [comment, marked as answer]
Hi @m-makhovyk 
Currently, there is no endpoint using API KEY for this **from Apple side**, so for now you need to use the session login.

**Reply by m-makhovyk** [reply]
Thanks!
